src/main/java/com/github/teamfusion/spyglassplus/common/entity/SpyglassStandEntity.java

```diff
@@ -111,20 +115,21 @@ public InteractionResult interact(Player player, InteractionHand hand) {
 				return InteractionResult.SUCCESS;
 			}
 		} else {
-			if (player.isShiftKeyDown()) {
+			if (this.getOwner() == null && player.isShiftKeyDown()) {
```

Comment:
Shouldn't this also check if the owner is the person doing it, not just that they're null?